modules/auxiliary/gather/memcached_extractor.rb

```diff
@@ -57,9 +57,10 @@ def enumerate_keys
       loop do
         sock.send("stats cachedump #{sid} #{max_keys}\r\n", 0)
```

Comment:
Is this meant to be outside of the loop?

Reply:
Hmm good point. Probably should be moved outside the loop, not sure why its inside the loop. Good catch

Reply:
Fixed with https://github.com/rapid7/metasploit-framework/pull/16704/commits/5b6d9538cd1a047c4d2cde171665224556329e00, thanks for pointing this out @adfoster-r7 !